<!doctype html>
<html lang="en">

		<head>
				<meta charset="utf-8">

				<title>SwipProteomics Slides</title>

				<meta name="description" content="SwipProteomics Job Talk">
				<meta name="author" content="Tyler W. A. Bradshaw">

				<meta name="apple-mobile-web-app-capable" content="yes">
				<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

				<meta name="viewport" content="width=device-width, initial-scale=1.0">

				<link rel="stylesheet" href="dist/reset.css">
				<link rel="stylesheet" href="dist/reveal.css">
				<link rel="stylesheet" href="lib/css/custom.css" id="theme">
				<link rel="stylesheet" href="plugin/highlight/zenburn.css">
		</head>

		<body>

				<!-- reveal.js slides -->
				<div class="reveal">

						<div class="slides">

								<!-- title slide -->
								<section>
										<em>Revealing Organization of the Neuronal Proteome with 
												Proximity and Spatial Proteomics</em>
										<img src="./lib/images/goodsell.png" alt="goodsell" style="width:50%">

										<h3>Tyler W. A. Bradshaw</h3>
										Neurobiology PhD
										<br>
										<small>Duke University</small>

										<div class='foot-left'>
												<a href="https://bit.ly/355Lvmg" target="_blank">Art by David Goodsell</a>
										</div>

										<div class='foot-right'>
												<a href="https://github.com/twesleyb/presentations/tree/master/swip" target="_blank">Slide Source Code</a>
										</div>

										<aside data-markdown class="notes" >
												# Hi, I'm Tyler Bradshaw!
										</aside>

								</section>

								<!-- objectives -->
								<section data-background-image="./lib/images/elife-image.png" 
										 data-background-size="fill" data-background-opacity=0.20>
										<h3>Objectives</h3>
										<ul>
												<li><a href="https://bit.ly/2RbEBZ8" alt="DOI: 10.7554/eLife.61590">Courtland and Bradshaw, eLife (2021):</a><em> Genetic disruption of WASHC4 drives endo-lysosomal dysfunction and cognitive-movement impairments</em></li>
												<br>
												<li>Resolving cellular space:</li>
												<ul>
														<li><em>Spatial Proteomics</em> (subcellular fractionation)</li>
														<li><em>Proximity Proteomics</em> (proximity labeling)</li>
												</ul>
										</ul>

										<aside data-markdown class="notes" >
												Transition: Both proximity and spatial proteomics hinge upon a fundamental biological principle...
										</aside>

								</section>

								<!-- example: cytoscape -->
								<section>
										<h2>Konigsberg Graph</h2>
										<div id="konigsberg-graph" class="cytoscape"></div>
								</section>

								<!-- background: fundamental biology -->
								<section>
										<h3>Cells are compartmentalized</h3>
										<figure>
												<img src="./lib/images/synapses.png" alt="synapses" style="width:100%">
												<figcaption>Cells are compartmentalized into functionally-defined membrane and non-membrane bound organelles.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												EM images: <a href="https://synapseweb.clm.utexas.edu/" target="_blank">SynapseWeb</a>
										</div>

										<aside data-markdown class="notes" >
												transition: we can use proteomics to identify the proteins that reside at these specialized subcellular compartments...
										</aside>

								</section>

								<!-- background: fundamental technology -->
								<section>
										<h4>Proteomics: Identifying and Quantifying proteins in Complex Biological Preparations</h4>
										<figure>
												<img src="./lib/images/ms-overview.png" alt="ms" style="width:90%">
												<figcaption><small>Normalization &rarr; Protein Summarization &rarr; <u>Statistical Testing</u></small></figcaption>
												<br>
												<div class='foot-right'>
														<a href="https://github.com/soderling-lab/tidyProt" target="_blank">soderling-lab/tidyProt</a>
												</div>
										</figure> 

										<aside data-markdown class="notes" >
												transition: proximity and spatial proteomics overview
										</aside>

								</section>

								<!-- background: overview -->
								<section>
										<h4>Resolving Subcellular Space with Proximity and Spatial Proteomics</h4>
										<figure>
												<img src="./lib/images/protms.png" alt="protms" style="width:100%">
												<figcaption>Proximity Proteomics (A) and Spatial Proteomics (B) define protein subcellular localization.</figcaption>
										</figure> 

										<aside data-markdown class="notes" >
												transition: we can use proteomics to identify the proteins that reside at these specialized subcellular compartments...
										</aside>

								</section>

								<!-- background: spatial proteomics -->
								<section>
										<h4>Spatial Proteomics: Resolving Subcellular Space by Subcellular Fractionation</h4>
										<figure>
												<img src="./lib/images/epsd.png" alt="epsd" style="width:80%">
												<figcaption>Subcellular fractions enriched for various biological fractions can be generated by centrifugation.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												SFARI ASD Genes: <a href="https://github.com/soderling-lab/getPPIs" target="_blank">soderling-lab/geneLists</a>
										</div>

										<aside data-markdown class="notes" >
												transition: while spatial proteomics leverages the fact that many organelles are enclosed in a plasma membrane...
										</aside>

								</section>

								<!-- background: proximity proteomics -->
								<section>
										<h4>Proximity Proteomics: Resolving Subcellular Space by Proximity Labeling</h4>

										<figure>
												<img src="./lib/images/gphn.png" alt="gphn-bioid" style="width:65%">
												<figcaption>Hard-to-reach subcellular compartments can be captured with BioID proximity proteomics.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												iPSD Proximity Proteome: <a href="https://bit.ly/3wO1CkG" target="_blank">Uezu, Kanak and Bradshaw <em>et al.</em>, 2016</a>
										</div>

										<aside data-markdown class="notes" >
												transition: both of these proteomics approaches use mass spectrometry to identify and quantify proteins...
										</aside>

								</section>

								<!-- intro: WASHC4 -->
								<section>
										<h3>SWIP, a Strumpellin and WASH Interacting Protein (gene: WASHC4)</h3>
										<figure>
												<img src="./lib/images/Swip.gif" alt="swip" style="width:50%">
												<figcaption>Autosomal recessive inheritance of the SWIP<sup>P1019R</sup> mutation is associated with intellectual disability.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/3qjL2qA" target="_blank">Ropers <em>et al.</em>, 2011</a>
										</div>

										<aside data-markdown class="notes" >
												transition: SWIP is encoded by the WASHC4 gene, and is a component of the WASH complex...
										</aside>

								</section>

								<!-- background: WASH Complex -->
								<section>
										<h3>SWIP is a Subunit of the Pentameric WASH Complex</h3>
										<figure>
												<img src="./lib/images/washcnew.png" alt="washc" style="width:30%">
												<figcaption>Mutations in WASHC4 (SWIP) and WASHC5 (Strumpellin) are associated with intellectual disability and spastic paraplegia.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/2SlQufW" target="_blank">Valdmanis <em>et al.</em>, 2007</a>
										</div>

										<aside data-markdown class="notes" >
												transition: work in yeast and heterologous cells...
										</aside>

								</section>

								<!-- background: WASH Complex Function -->
								<section>
										<h3>An Intracellular Trafficking Hub</h3>
										<img class="r-stretch" src="./lib/images/trafficking.png">
										<p>The WASH complex acts upstream of Arp2/3 actin polymerization to direct early endosome maturation.</p>
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/3gSvDsV" target="_blank">Derivery <em>et al.</em>, 2009</a>
										</div>

										<aside data-markdown class="notes" >
												transition: to date, no one has studied the function of WASHC4 in the brain...
										</aside>

								</section>

								<!-- transition: paper -->
								<section data-background-image="./lib/images/paper.png" 
										 data-background-size="contain" data-background-opacity=0.20>
										<h2 class="r-fit-text">Key Questions</h2>
										<ul>
												<li>What is the function of WASHC4 in the brain?</li>
												<li>What is the affect of the P1019R mutation on the neuronal proteome?</li>
										</ul>

										<aside data-markdown class="notes" >
												transition: we addressed these questions using proteomics...
										</aside>

								</section>

								<!-- results: WASH BioID -->
								<section>
										<h3>WASH <em>in vivo</em> BioID</h3>
										<figure>
												<img src="./lib/images/bioid-ihc.png" alt="wash-bioid-ihc" style="width:70%">
										</figure> 
										<p><b>Negative control:</b> Soluble BioID2</p>

										<aside data-markdown class="notes" >
												transition: I performed analysis in order to identify proteins that are significantly enriched...
										</aside>

								</section>

								<!-- results: WASH Proximity Proteome -->
								<section>
										<h3>The WASH Proximity Proteome</h3>
										<img class="r-stretch" src="./lib/images/wash-bioid.png">
										<p>Identification of the <em>in vivo</em> WASH proximity proteome supports its role in endosomal trafficking.</p>

										<aside data-markdown class="notes" >
												transition: to further interrogate the function of WASHC4 in the brain...
										</aside>

								</section>

								<!-- results: WASHC4 mutant mouse -->
								<section>
										<h3>A WASHC4<sup>P1019R</sup> Mutant Mouse</h3>
										<figure>
												<img src="./lib/images/washc4mouse.png" alt="mut-mouse" style="width:100%">
												<figcaption>The P1019R missense mutation results in decreased Strumpellin and WASH1 protein.</figcaption>
										</figure> 

										<aside data-markdown class="notes" >
												transition: we used this mutant mouse to study the consequences of the P1019R mutation on the neuronal proteome...
										</aside>

								</section>

								<!-- results: WASH Spatial Proteome -->
								<section>
										<h3>SWIP Spatial Proteomics</h3>
										<figure>
												<img src="./lib/images/spatprot.png" alt="spatial-proteomics" style="width:100%">
												<figcaption>Profiling protein abundance across seven biological fractions (BioFractions).</figcaption>
												<br>
												<div class='foot-right'>
														LOPIT-DC: <a href="https://go.nature.com/3gQNKjW" target="_blank">Geladaki <em>et al.</em>, 2019</a>
												</div>

												<aside data-markdown class="notes" >
														transition: analysis of these data presents several challenges...
												</aside>

								</section>

								<!-- EMPHASIS: challenges -->
								<section data-background-image="./lib/images/spatprot-network.png" 
										 data-background-size="contain" data-background-opacity=0.20>
										<h3>Challenges inherit in analyzing spatial proteomics</h3>
										<br>
										<div class="container">
												<div class="col">
														<h4>Community Detection</h4>
														<em>Identify Groups of Covarying Proteins</em>
												</div>
												<div class="col">
														<h4>Statistical Inference</h4>
														<em>Identify (Protein and Module) Changes</em>
												</div>
										</div>
										<br><br>
										<div class='foot-left'>
												pRoloc: <a href="https://bit.ly/3wYPayA" target="_blank">Breckels, Gatto, Crook <em>et al.</em>, 2014, 2016, 2019</a>
										</div>

										<aside data-markdown class="notes" >
												transition: my solution to these problems...
										</aside>

								</section>

								<!-- EMPHASIS: Solutions -->
								<section data-background-image="./lib/images/spatprot-network.png" 
										 data-background-size="contain" data-background-opacity=0.20>
										<h3>Solutions to analysis of spatial proteomics</h3>
										<br>
										<div class="container">
												<div class="col">
														<h4>Community Detection</h4>
														<em>Network Enhancement + Leiden Algorithm</em>
														<br>
														<small>neten: <a href="https://go.nature.com/3gT86Jv" target="_blank">Wang <em>et al.</em> 2018</a></small>
														<small>leidenalg: <a href="https://go.nature.com/3gSl75U" target="_blank">Traag <em>et al.</em> 2019</a></small>
												</div>

												<div class="col">
														<h4>Statistical Inference</h4>
														<em>Linear Mixed Effects Models</em>
														<br><br>
														<small>lme4: <a href="https://bit.ly/2TTIl2F" target="_blank">Bates <em>et al.</em>, 2021</a></small>
														<small>t-statistic (lmerTest): <a href="https://bit.ly/3gYRnTX" target="_blank">Kuznetsova <em>et al.</em>, 2017</a></small>
												</div>
										</div>

										<aside data-markdown class="notes" >
												transition: I use linear models to assess differential abundance at two different resolutions...
										</aside>

								</section>


								<!-- overview: spatprot network -->
								<section data-background-image="./lib/images/spatprot-network.png" 
										 data-background-size="contain" data-background-opacity=0.90>
										<h2 class="r-fit-text">Neten + Leiden (Surprise)</h2>
										<p>Nodes = 5,869</p>
										<p>k = 49</p>
										<br>
										<p><small>Endosome (M38) | Lysosome (M36) | ER (M6) | Synapse (M27)</small></p>

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Endosomal Module -->
								<section>
										<h3>Significant Disruption of an Endosomal Module Containing SWIP</h3>
										<figure>
												<img src="./lib/images/endosome.png" alt="m38" style="width:70%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Endosome ICC -->
								<section>
										<h3>WASHC4 Mutant Neurons Exhibit Altered Endosome Structures</h3>
										<figure>
												<img src="./lib/images/endosome-icc.png" alt="endosome-icc" style="width:75%">
												<figcaption>Increased early endosome volume is consistent with WASH-complex loss-of-function.</figcaption>
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Lysosome Module -->
								<section>
										<h4>Disruption of a Lysosomal Protein Network</h4>
										<figure>
												<img src="./lib/images/lysosome-2.png" alt="bioid" style="width:80%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Lipofuscin -->
								<section>
										<h3>Lipofuscin Accumulation in SWIP<sup>P1019R</sup> Mutant Brain</h3>
										<figure>
												<img src="./lib/images/lyso-tem-2.png" alt="lyso-tem" style="width:100%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: ER Stress Module -->
								<section>
										<h3>Disruption of an ER Stress Associated Module</h3>
										<figure>
												<img src="./lib/images/m6.png" alt="m6" style="width:80%">
												<figcaption>Increased ER stress proteins: e.g. HSPA5, PDIA3, PDIA4, PDIA6, and DNAJC3</figcaption>
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Neurodegeneration -->
								<section>
										<h3>Evidence of Neurodegeneration in SWIP<sup>P1019R</sup> Adult Motor Cortex</h3>
										<figure>
												<img src="./lib/images/caspase.png" alt="caspase" style="width:60%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Synapse -->
								<section>
										<h4>Decreased abundance of a Synapse Module</h4>
										<figure>
												<img src="./lib/images/synapse-2.png" alt="synapse" style="width:80%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>


								<!-- results: Decreased Synapse Number -->
								<section>
										<h3>Decreased Synapse Number in SWIP<sup>P1019R</sup> Adult Motor Cortex</h3>
										<figure>
												<img src="./lib/images/synapse-ihc.png" alt="synapse-ihc" style="width:50%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>


								<!-- results: aged-motor -->
								<section>
										<h3>Motor Function Decline with Age</h3>
										<figure>
												<img src="./lib/images/treadscan.png" alt="varpart" style="width:100%">
										</figure> 
										<p><b>Not shown:</b> rotorod deficit, impaired fear conditioning</p>

										<aside data-markdown class="notes" >
												transition: strikingly, ...
										</aside>

								</section>

								<!-- results: human-motor -->
								<section>
										<h3>Impaired motor behaviors in SWIP<sup>P1019R</sup> Patient Cohort</h3>
										<figure>
												<img src="./lib/images/hs-motor.png" alt="hs-motor" style="width:100%">
										</figure> 

										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- Conclusions -->
								<section>
										<h3>Updated Model: Parallels with Neurodegeneration Disorders</h3>
										<figure>
												<img src="./lib/images/model.png" alt="image" style="width:100%">
										</figure> 
								</section>

								<!-- Future Directions -->
								<section>
										<h2>Future Directions</h2>
										<p>Resources for reproducible proteomics research.</p>
										<div class="container">
												<div class="col">
														<figure>
																<img src="./lib/images/github.png" alt="image" style="width:100%">
														</figure> 
												</div>
												<div class="col">
														<figure>
																<img src="./lib/images/multi-graph.png" alt="image" style="width:100%">
														</figure> 
												</div>
										</div>
										<p>Generating high-resolution cellular maps with the Leiden algorithm.</p>
								</section>

								<!-- Acknowledgments -->
								<section>
										<h2>Acknowledgments</h2>
										<ul>
												<li><b>Collaborator:</b> Dr. Jamie Courtland, MD, PhD</li>
												<li><b>Thesis Lab:</b> Dr. Scott Soderling, PhD</li>
												<li><b>Proteomics Core:</b> Greg Waitt and Erik Soderblom</li>
												<li><b>Funding:</b> (NIH F31 5F31NS113738-03)</li>
										</ul>
								</section>

								<!-- Questions -->
								<section data-background-image="./lib/images/spatprot-network.png" 
										 data-background-size="fill" data-background-opacity=0.20>
										<h2 class="r-fit-text">Questions</h2>
										<br>
										<div class='foot-right'>
												email: <a href="mailto: twesleyb10@gmail.com">twesleyb10@gmail.com</a>
										</div>
								</section>

								<!-- Extras -->
								<section>
										<h2>Extra Slides</h2>
								</section>

								<!-- transition: Spatial Proteomics Network -->
								<section>
										<h3>The Swip Spatial Proteome</h3>
										<img class="r-stretch" src="./lib/images/swip-spatprot-netw.png">
								</section>

								<!-- intro: Rogdi -->
								<section>
										<h3>Rogdi, Rav2, and the RAVE Complex</h3>
										<figure>
												<img src="./lib/images/rogdi-rav2.png" alt="rogdi-structure" style="width:50%">
												<figcaption>Structural alignment of Rogdi and Rav2.</figcaption>
										</figure> 
										<br>
										<div class='foot-right'>
												Rogdi crystal structure: <a href="https://go.nature.com/3d6jpf1" target="_blank">Lee <em>et al.</em>, 2017</a>
										</div>
								</section>

								<!-- result: SV m38 -->
								<section>
										<h3>A SV Module Containing Rogdi</h3>
										<figure>
												<img src="./lib/images/m35.png" alt="m35" style="width:65%">
												<figcaption><b>Hypothesis:</b> Rogdi localizes to presynaptic SV.</figcaption>
										</figure> 
								</section>

								<!-- edgeR GOF -->
								<section>
										<h3>Goodness-of-fit of the <code>edgeR</code> model for TMT Proteomics</h3>
										<figure>
												<img src="./lib/images/gof.png" alt="gof" style="width:100%">
												<figcaption>Quantile-Quantile plots evaluating the goodness of fit of the NB model for TMT proteomics data.</figcaption>
										</figure> 
								</section>

								<!-- network enhancment -->
								<section>
										<h3>Network Enhancement Facilitates Community Detection</h3>
										<figure>
												<img src="./lib/images/neten.png" alt="neten" style="width:50%">
										</figure> 
								</section>

								<!-- variance partition -->
								<section>
										<h3>Partitioning the Variance</h3>
										<figure>
												<img src="./lib/images/varpart.png" alt="varpart" style="width:75%">
										</figure> 
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/35OtYiG" target="_blank">Hoffman <em>et al.</em> 2016, 2019</a>
										</div>
								</section>

								<!-- statistics -->
								<section>
										<h2>Extending Linear Models with Mixed Effects</h2>

										<div class="container">

												<div class="col">
														\[\begin{aligned}
														Y_{p} = X\beta_{p} + \epsilon_{p} \\ % lm
														Y_{p} =  Z\alpha_{p} + X\beta_{p} + \epsilon_{p} \\ % lmer
														\epsilon_{p} \stackrel{iid}{\sim} N(0,\sigma^2) \\ % e = N(0,s2)
														\alpha_{p} \stackrel{iid}{\sim} N(0,\sigma_{Z}^2) \\ % a = N(0,s2)
														\end{aligned} \]
												</div>

												<div class="col">
														\[\begin{aligned}
														H_0: l^T * \beta = 0 \\
														t = \frac{l^T \hat{\beta}}{\sqrt{l \sigma^2 \hat{V} l^T}} \\ % t-stat
														\end{aligned} \]
												</div>
										</div>
								</section>

								<!-- leidenalg: cpm -->
								<section>
										<h3>Structure in Graphs at Multiple Resolutions</h3>
										<div class="container">

												<div class="col">
														<figure>
																<img src="./lib/images/konigsberg-cropped.gif" alt="konigsberg-gif" style="width:75%">
														</figure> 
												</div>

												<div class="col">
														<figure>
																<img src="./lib/images/cpm.png" alt="cpm" style="width:58%">
														</figure> 
												</div>
										</div>
										<p><code>Konigsberg < Kaliningrad < Russia</code></p>
								</section>

						</div>
				</div>
				<!-- end slides -->

				<!-- reveal.js src -->
				<script src="dist/reveal.js"></script>
				<script src="plugin/math/math.js"></script>
				<script src="plugin/zoom/zoom.js"></script>
				<script src="plugin/notes/notes.js"></script>
				<script src="plugin/search/search.js"></script>
				<script src="plugin/markdown/markdown.js"></script>
				<script src="plugin/highlight/highlight.js"></script>
				<script>

						Reveal.initialize({
								// slide settings
								controls: true,
								progress: true,
								center: true,
								hash: true,
								slideNumber: true,
								// cytoscape adjustments
								width: '100%',
								height: '100%',
								margin: 0,
								minScale: 1,
								maxScale: 1,
								// plugins
								plugins: [RevealZoom, RevealNotes, RevealSearch, RevealMarkdown, RevealHighlight, RevealMath]
						});

				</script>

				<!-- cytoscape.js src -->
				<script src="lib/js/cytoscape/cytoscape.min.js"></script>
				<script src="lib/graphs/konigsberg/data.js"></script>
				<script src="lib/graphs/konigsberg/graph.js"></script>

		</body>
</html>
