<!doctype html>
<html lang="en">

		<head>
				<meta charset="utf-8">

				<title>SwipProteomics Slides</title>

				<meta name="description" content="SwipProteomics Job Talk">
				<meta name="author" content="Tyler W. A. Bradshaw">

				<meta name="apple-mobile-web-app-capable" content="yes">
				<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

				<meta name="viewport" content="width=device-width, initial-scale=1.0">

				<link rel="stylesheet" href="dist/reset.css">
				<link rel="stylesheet" href="dist/reveal.css">
				<link rel="stylesheet" href="plugin/highlight/zenburn.css">
				<link rel="stylesheet" href="plugin/pointer/pointer.css">

				<link rel="stylesheet" href="lib/css/custom.css" id="theme">
		</head>

		<body>

				<!-- reveal.js slides -->
				<div class="reveal">

						<div class="slides">


								<!-- title slide -->
								<section>
										<em>Revealing Organization of the Neuronal Proteome with
												Proximity and Spatial Proteomics</em>
										<img class="r-stretch" src="./lib/images/goodsell.png" alt="goodsell">

										<h3>Tyler W. A. Bradshaw</h3>
										Neurobiology PhD
										<br>
										<small>Duke University</small>

										<div class='foot-left'>
												<a href="https://bit.ly/355Lvmg" target="_blank">Art by David Goodsell</a>
										</div>

										<div class='foot-right'>
												<a href="https://github.com/twesleyb/presentations/tree/master/SwipProteomics" target="_blank">Slide Source Code</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												# Hi, I'm Tyler Bradshaw!
												Today I want to tell you about
												my dissertation work which has
												been to reveal organization of
												the neuronal proteome using two
												types of proteomics: proximity
												and spatial proteomics.
										</aside>

								</section>

								<!-- objectives -->
								<section data-background-image="./lib/images/elife-image.png" 
				 data-background-size="fill" data-background-opacity=0.20>
										<h3>Objectives</h3>
										<ul>
												<li><a href="https://bit.ly/2RbEBZ8" alt="DOI: 10.7554/eLife.61590">Courtland and Bradshaw, eLife (2021):</a><em> Genetic disruption of WASHC4 drives endo-lysosomal dysfunction and cognitive-movement impairments in mice and humans.</em></li>
												<br>
												<li><b>Resolving cellular space:</b></li>
												<ul>
														<li><em>Spatial Proteomics</em> (subcellular fractionation)</li>
														<li><em>Proximity Proteomics</em> (proximity labeling)</li>
												</ul>
										</ul>

										<!-- notes -->
										<aside data-markdown class="notes" >
												My objective today is to tell
												you about my recent
												publication, co-authored with
												Dr. Jamie Courtland, another
												graduate student in my thesis
												lab. 

												Transition: Both proximity and
												spatial proteomics hinge upon a
												fundamental biological
												principle... cells are
												compartmentalized.
										</aside>

								</section>

								<!-- background: overview -->
								<section>
										<h4>Proximity and Spatial Proteomics define protein localization</h4>
										<img class="r-stretch" src="./lib/images/protms-netw.png" alt="protms">

										<aside data-markdown class="notes" >
												transition: we can use proteomics to identify the proteins that reside at these specialized subcellular compartments...
										</aside>
								</section>

								<!-- background: fundamental biology -->
								<section>
										<h3>Cells are compartmentalized</h3>
										<img class="r-stretch" src="./lib/images/synapses.png" alt="synapses">
										<p>Cells are compartmentalized into functionally-defined membrane and non-membrane bound organelles.</p>
										<br>
										<div class='foot-right'>
												EM images: <a href="https://synapseweb.clm.utexas.edu/" target="_blank">SynapseWeb</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: we can use
												proteomics to identify the
												proteins that reside at these
												specialized subcellular
												compartments...
										</aside>

								</section>

								<!-- background: fundamental technology -->
								<section>
										<h4>Proteomics: Identifying and Quantifying proteins in Complex Biological Preparations</h4>
										<img class="r-stretch" src="./lib/images/ms-overview.png" alt="ms">
										<p>Normalization &rarr; Protein Summarization &rarr; 
										<u>Statistical Testing</u></p>
										<br>
										<div class='foot-right'>
												statistical testing: <a href="https://github.com/soderling-lab/tidyProt" target="_blank">soderling-lab/tidyProt</a>
										</div>
										</figure> 

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: spatial and
												proximity proteomics aim to
												define where a protein
												functions in the cell.
										</aside>

								</section>

								<!-- background: spatial proteomics -->
								<section>
										<h4>Spatial Proteomics: Resolving Subcellular Space by Fractionation</h4>
										<img class="r-stretch" src="./lib/images/epsd.png" alt="epsd">
										<p>Subcellular fractions enriched for specific biological fractions can be generated by centrifugation.</p>
										<br>
										<div class='foot-right'>
												SFARI ASD Genes: <a href="https://github.com/soderling-lab/geneLists" target="_blank">soderling-lab/geneLists</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: while spatial
												proteomics leverages the fact
												that many organelles are
												enclosed in a plasma membrane
												and can therefore be enriched
												by differential centrifugation
												...
										</aside>

								</section>

								<!-- assembling covariation networks -->
								<section>
										<h3>Assembling Protein Covariation Networks</h3>
										<img class="r-stretch" src="./lib/images/cor-mat-ppi.png" alt="prot-covar">
										<p>The abundance of proteins that function and interact together tend to be highly correlated.</p>
								</section>


								<!-- background: proximity proteomics -->
								<section>
										<h4>Proximity Proteomics: Resolving Subcellular Space by Proximity Labeling</h4>

										<img class="r-stretch" src="./lib/images/gphn.png" alt="gphn-bioid">
										<p>Hard-to-reach subcellular compartments can be captured with BioID proximity proteomics.</p>
										<br>
										<div class='foot-right'>
												iPSD Proximity Proteome: <a href="https://bit.ly/3wO1CkG" target="_blank">Uezu, Kanak and Bradshaw <em>et al.</em>, 2016</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: both of these
												proteomics approaches use mass
												spectrometry to identify and
												quantify proteins...
										</aside>

								</section>


								<!-- background: structure and function in biological networks -->
								<section>
										<h3>Structure and Function in Biological Networks</h3>
										<img class="r-stretch" src="./lib/images/ipsd-modules.png" alt="ipsd-modules">
										<br><br>
										<div class='foot-right'>
												<a href="https://github.com/soderling-lab/getPPIs" target="_blank">soderling-lab/getPPIs</a>
										</div>
								</section>


								<!-- intro: community detection -->
								<section>
										<h4>Two Routes to Community Detection</h4>
										<div class="container">
												<div class="col">
														<img class="r-stretch" src="./lib/images/svm.png" alt="svm">
														<p><b>Supervised:</b> <small><em>e.g.</em> Support Vector Machine<br>(SVM)</small></p>
												</div>
												<div class="col">
														<img class="r-stretch" src="./lib/images/unsupervised.png" alt="unsupervised">
														<p><b>Unsupervised:</b> <small><em>e.g.</em> Leiden Algorithm<br>(Quality Optimization)</small></p>
												</div>
										</div>
								</section>

								<!-- intro: WASHC4 -->
								<section>
										<h4>SWIP, a Strumpellin and WASH Interacting Protein (gene: WASHC4)</h4>
										<img class="r-stretch" src="./lib/images/Swip.gif" alt="gif">
										<p>Autosomal recessive inheritance of the SWIP<mysup>P1019R</mysup> mutation is associated with intellectual disability.</p>
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/3qjL2qA" target="_blank">Ropers <em>et al.</em>, 2011</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: SWIP is encoded by
												the WASHC4 gene, and is a
												component of the WASH
												complex...
										</aside>

								</section>

								<!-- background: WASH Complex -->
								<section>
										<h3>SWIP is a Subunit of the Pentameric WASH Complex</h3>
										<img class="r-stretch" src="./lib/images/washcnew.png" alt="washc">
										<p>Mutations in WASHC4 (SWIP) and WASHC5 (Strumpellin) are associated with intellectual disability and spastic paraplegia.</p>
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/2SlQufW" target="_blank">Valdmanis <em>et al.</em>, 2007</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: work in yeast and
												heterologous cells...
										</aside>

								</section>

								<!-- background: WASH Complex Function -->
								<section>
										<h4>An Intracellular Trafficking Hub</h4>
										<img class="r-stretch" src="./lib/images/trafficking.png" alt="hub">
										<p>The WASH complex acts upstream of Arp2/3 actin polymerization to direct endosome maturation.</p>
										<br>
										<div class='foot-right'>
												<a href="https://bit.ly/3gSvDsV" target="_blank">Derivery <em>et al.</em>, 2009</a>
										</div>

										<aside data-markdown class="notes" >
												transition: to date, no one has studied the function of WASHC4 in the brain...
										</aside>

								</section>

								<!-- transition: paper -->
								<section data-background-image="./lib/images/paper.png" 
				 data-background-size="contain" data-background-opacity=0.20>
										<h2 class="r-fit-text">Key Questions</h2>
										<ul>
												<li>What is the function of WASHC4 in the brain?</li>
												<li>How does the P1019R mutation affect the neuronal proteome?</li>
										</ul>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: we addressed these
												questions using proteomics...
										</aside>

								</section>

								<!-- results: WASH BioID -->
								<section>
										<h3>WASH <em>in vivo</em> BioID</h3>
										<img class="r-stretch" src="./lib/images/bioid-ihc.png" alt="wash-bioid-ihc">
										<p><b>Negative control:</b> Soluble BioID2</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: I performed
												analysis in order to identify
												proteins that are significantly
												enriched...
										</aside>

								</section>

								<!-- results: WASH Proximity Proteome -->
								<section>
										<h3>The WASH Proximity Proteome</h3>
										<img class="r-stretch" src="./lib/images/wash-bioid.png" alt="bioid-netw">
										<p>Identification of the <em>in vivo</em> WASH proximity proteome supports its role in endosomal trafficking.</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: to further
												interrogate the function of
												WASHC4 in the brain...
										</aside>

								</section>

								<!-- results: WASHC4 mutant mouse -->
								<section>
										<h3>A WASHC4<sup>P1019R</sup> Mutant Mouse</h3>
										<img class="r-stretch" src="./lib/images/washc4mouse.png" alt="mut-mouse">
										<p>The P1019R missense mutation results in decreased Strumpellin and WASH1 protein.</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: we used this mutant
												mouse to study the consequences
												of the P1019R mutation on the
												neuronal proteome...
										</aside>

								</section>

								<!-- results: WASH Spatial Proteome -->
								<section>
										<h3>SWIP Spatial Proteomics</h3>
										<img class="r-stretch" src="./lib/images/spatprot-profile.png" alt="spatial-proteomics">
										<p>Profiling protein abundance across seven biological fractions (BioFractions).</p>
										<br>
										<div class='foot-right'>
												LOPIT-DC: <a href="https://go.nature.com/3gQNKjW" target="_blank">Geladaki <em>et al.</em>, 2019</a>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: analysis of these
												data presents several
												challenges...
										</aside>

								</section>

								<!--pRoloc: <a href="https://bit.ly/3wYPayA" target="_blank">Breckels, Gatto, Crook <em>et al.</em>, 2014, 2016, 2019</a> -->

								<!-- EMPHASIS: Solutions -->
								<section data-background-image="./lib/images/swip-spatprot-netw.png" 
				 data-background-size="contain" data-background-opacity=0.20>
										<h3>Challenges inherent in analyzing spatial proteomics</h3>
										<br>
										<div class="container">
												<div class="col">
														<h4>Community Detection</h4>
														<em>Network Enhancement + Leidenalg (Surprise)</em>
														<br>
														<small><code>neten</code>: <a href="https://go.nature.com/3gT86Jv" target="_blank">Wang <em>et al.</em> 2018</a></small>
														<small><code>leidenalg</code>: <a href="https://go.nature.com/3gSl75U" target="_blank">Traag <em>et al.</em> 2019</a></small>
												</div>

												<div class="col">
														<h4>Statistical Inference</h4>
														<em>Linear Mixed-Effects Models</em>
														<br>
														<small><code>lme4</code>: <a href="https://bit.ly/2TTIl2F" target="_blank">Bates <em>et al.</em>, 2021</a></small>
														<small>t-statistic: <a href="https://bit.ly/3gYRnTX" target="_blank">Kuznetsova <em>et al.</em>, 2017</a></small>
												</div>
										</div>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: I use linear models
												to assess differential
												abundance at two different
												resolutions...
										</aside>
								</section>

								<!-- overview: spatprot network -->
								<section>
										<h2 class="r-fit-text">Swip Spatial Proteome</h2>
										<img class="r-stretch" src="lib/images/swip-spatprot-netw.png">
										<p><small>(Nodes = 5,869 | Modules = 49)</small></p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- two levels of analysis -->
								<section>
										<h4>Module-level Analysis Powers Biological Insight</h4>
										<img class="r-stretch" src="lib/images/washc4-prot-module.png">
										<p>
										<b>module-level model:</b> <code>log2(Abundance) ~ 0 + Condition + (1|Protein)</code>
										</p>
								</section>

								<!-- NS compartments -->
								<section>
										<h4>No change in six Compartments Identified by Spatial Proteomics</h4>
										<img class="r-stretch" src="./lib/images/organelles.png" alt="ns-compartments">


								</section>

								<!-- results: Endosomal Module -->
								<section>
										<h4>Significant Disruption of an Endosomal Module Containing SWIP</h4>
										<img class="r-stretch" src="lib/images/endosome.png">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Endosome ICC -->
								<section>
										<h4>WASHC4 Mutant Neurons Exhibit Altered Endosome Structures</h4>
										<img class="r-stretch" src="./lib/images/endosome-icc.png" alt="endosome-icc">
										<p>Increased early endosome volume is consistent with WASH-complex loss-of-function.</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Lysosome Module -->
								<section>
										<h4>Disruption of a Lysosomal Protein Network</h4>
										<img class="r-stretch" src="./lib/images/lysosome-2.png" alt="bioid">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Lipofuscin -->
								<section>
										<h3>Lipofuscin Accumulation in SWIP<sup>P1019R</sup> Mutant Brain</h3>
										<img class="r-stretch" src="./lib/images/lyso-tem.png" alt="lyso-tem">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- results: Neurodegeneration -->
								<section>
										<h4>Evidence of Neurodegeneration in SWIP<sup>P1019R</sup> Adult Motor Cortex</h4>
										<img class="r-stretch" src="./lib/images/caspase.png" alt="caspase">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>


								<!-- results: aged-motor -->
								<section>
										<h4>Motor Function Decline with Age</h4>
										<div class="container">
												<div class="col">
														<figure>
																<figcaption>Treadscan Analysis</figcaption>
																<!--<video data-autoplay src="./lib/images/treadscan.mp4" style="width:100%"></video>-->
																<img src="./lib/images/treadscan.gif" style="width:100%">
														</figure>
												</div>
												<div class="col">
														<figure>
																<img src="./lib/images/treadscan.png" alt="treadscan" style="width:80%">
														</figure> 
												</div>
										</div>
										<p>WASHC4<sup>P1019R</sup> mutant mice exhibit motor function decline with age.</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition: strikingly, ...
										</aside>

								</section>

								<!-- results: human-motor -->
								<section>
										<h3>Impaired motor behaviors in SWIP<sup>P1019R</sup> Patient Cohort</h3>
										<img class="r-stretch" src="./lib/images/hs-motor.png" alt="hs-motor">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- Conclusions -->
								<section>
										<h4>Conclusions and Parallels with Neurodegenerative Disorders</h4>
										<img class="r-stretch" src="./lib/images/model.png" alt="image">
								</section>

								<!-- Future Directions -->
								<section>
										<h2>Future Directions</h2>
										<p>Resources for reproducible proteomics research.</p>
										<div class="container">
												<div class="col">
														<figure>
																<img src="./lib/images/github.png" alt="image" style="width:100%">
														</figure> 
												</div>
												<div class="col">
														<figure>
																<img src="./lib/images/multi-graph.png" alt="image" style="width:100%">
														</figure> 
												</div>
										</div>
										<p>Generating high-resolution cellular maps with the Leiden algorithm.</p>
								</section>

								<!-- Acknowledgments -->
								<section>
										<h2>Acknowledgments</h2>
										<ul>
												<li><b>Collaborator:</b> Jamie Courtland</li>
												<li><b>Thesis Lab:</b> Scott Soderling</li>
												<li><b>Proteomics Core:</b> Greg Waitt, Tricia Ho, and Erik Soderblom</li>
												<li><b>Funding:</b> (NIH F31 5F31NS113738-03)</li>
										</ul>
								</section>

								<!-- Questions -->
								<section data-background-image="./lib/images/swip-spatprot-netw.png" 
				 data-background-size="fill" data-background-opacity=0.20>
										<h2 class="r-fit-text">Questions</h2>
										<br>
										<div class='foot-right'>
												email: <a href="mailto: twesleyb10@gmail.com">twesleyb10@gmail.com</a>
										</div>
								</section>

								<!-- extras -->
								<section>
										<h2 class="r-fit-text">Extra Slides</h2>
								</section>

								<!-- variance partition -->
								<section>
										<h3>Partitioning the Variance</h3>
										<img class="r-stretch" src="./lib/images/pca-varpart.png" alt="varpart">
										<br><br>
										<div class='foot-right'>
												VariancePartition: <a href="https://bit.ly/35OtYiG" target="_blank">Hoffman <em>et al.</em> 2016 & 2019</a>
										</div>
								</section>

								<!-- network enhancement -->
								<section>
										<h4>Network Enhancement Facilitates Community Detection</h4>
										<img class="r-stretch" src="./lib/images/neten.png" alt="neten">
										<br><br>
										<div class='foot-right'>
												R implementation: <a href="https://github.com/soderling-lab/neten" target="_blank">soderling-lab/neten</a>
										</div>
								</section>


								<!-- intro: leiden alg table -->
								<section data-markdown>
										 <textarea data-template>
												#### The Leiden Algorithm Functions to Maximize a Quality Statistic
												| Quality | Multi-Resolution | Weighted | Signed | Reference |
												|:--------|:----------------:|:--------:|:------:|:----------|
												| Modularity | False | True | False | Newman _et al._, 2004 |
												| RBER | True | True | False | Reichardt _et al._, 2006 |
												| RBConfiguration | True | True | False | Reichardt _et al._, 2006 |
												| CPM | True | True | True | Traag _et al._, 2011 |
												| Significance | False | False | False | Traag _et al._, 2013 |
												| Surprise | False | True | False | Traag _et al._, 2015 |

												Leiden Algorithm: [Traag _et al._, 2019](https://www.nature.com/articles/s41598-019-41695-z)
										 </textarea>
								</section>


								<!-- results: ER Stress Module -->
								<section>
										<h3>Disruption of an ER Stress Associated Module</h3>
										<img class="r-stretch" src="./lib/images/m6.png" alt="m6">
										<p>Increased ER stress proteins: e.g. HSPA5, PDIA3, PDIA4, PDIA6, and DNAJC3</p>

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- subtle sig compartments -->
								<section>
										<h3>Compartments with subtle changes in SWIP Mutant Mice</h3>
										<img class="r-stretch" src="./lib/images/cyto-mito.png" alt="sig-compartments">
										<p>Compartments enriched for Geladaki <em>et al.,</em> (2019) marker proteins.</p>
								</section>



								<!-- results: alt Endosome Module -->
								<section>
										<h3>Disruption of an Endosomal Module</h3>
										<img class="r-stretch" src="./lib/images/m22.png" alt="m22">
								</section>

								<!-- results: Synapse -->
								<section>
										<h4>Decreased abundance of an Excitatory Postsynapse Module</h4>
										<img class="r-stretch" src="./lib/images/synapse-2.png" alt="synapse">

										<!-- notes -->
										<aside data-markdown class="notes" >
												transition:
										</aside>

								</section>

								<!-- intro: konigsberg problem -->
								<section>
										<h2>Konigsberg Problem</h2>
										<img class="r-stretch" src="./lib/images/konigsberg.png" alt="konigsberg">
										<p>A view of the modern city of Kaliningrad, Russia.</p>
								</section>

								<!--  konigsberg graph -->
								<section>
										<h3>Euler 1735: Konigsberg Graph</h3>
										<div class="fig-container r-stretch" data-file="lib/graphs/konigsberg/index.html"></div>
								</section>

								<!-- leidenalg -->
								<section>
										<h3>The Leiden Algorithm</h3>
										<img class="r-stretch" src="./lib/images/leiden.png" alt="leiden">
										<br><br>
										<div class='foot-right'>
												Adapted from: <a href="https://go.nature.com/3gSl75U" target="_blank">Traag <em>et al.</em> 2019</a>
										</div>
								</section>

								<!-- leidenalg: cpm -->
								<section>
										<h3>Structure in Graphs at Multiple Resolutions</h3>
										<div class="container">

												<div class="col">
														<img class="r-stretch" src="./lib/images/konigsberg-cropped.gif" alt="konigsberg-gif" style="width:75%">
												</div>

												<div class="col">
														<img class="r-stretch" src="./lib/images/cpm.png" alt="cpm" style="width:58%">
												</div>
										</div>
										<p><code>Konigsberg < Kaliningrad < Europe</code></p>
										<br>
										<div class='foot-right'>
												Leiden Algorithm: <a href="https://go.nature.com/3gSl75U" target="_blank">Traag <em>et al.</em> 2019</a>
										</div>
								</section>


								<!-- permutation approach -->
								<section>
										<h4>Permutation testing to identify changes in network topology</h4>
										<img class="r-stretch" src="./lib/images/permutations.png" alt="netrep">
										<br><br>
										<div class='foot-right'>
												NetRep: <a href="https://bit.ly/3qQpuSy" target="_blank">Ritchie <em>et al.</em> 2016</a>
										</div>
								</section>

								<!-- statistics -->
								<section>
										<h2>Extending Linear Models with Mixed Effects</h2>

										<div class="container">

												<div class="col">
														\[\begin{aligned}
														Y_{p} = X\beta_{p} + \epsilon_{p} \\ % lm
														Y_{p} =  Z\alpha_{p} + X\beta_{p} + \epsilon_{p} \\ % lmer
														\epsilon_{p} \stackrel{iid}{\sim} N(0,\sigma^2) \\ % e = N(0,s2)
														\alpha_{p} \stackrel{iid}{\sim} N(0,\sigma_{Z}^2) \\ % a = N(0,s2)
														\end{aligned} \]
												</div>

												<div class="col">
														\[\begin{aligned}
														H_0: l^T * \beta = 0 \\
														t = \frac{l^T \hat{\beta}}{\sqrt{l \sigma^2 \hat{V} l^T}} \\ % t-stat
														\end{aligned} \]
												</div>
										</div>
								</section>

								<!-- edgeR GOF -->
								<section>
										<h3>Goodness-of-fit of the <code>edgeR</code> model for TMT Proteomics</h3>
										<img class="r-stretch" src="./lib/images/gof.png" alt="gof">
										<p>Quantile-Quantile plots evaluating the goodness of fit of the NB model for TMT proteomics data.</p>
								</section>

								<!-- Rogdi Mutations -->
								<section>
										<h4>Loss-of-function mutations in Rogdi are associated with Kohlschütter-Tönze Syndrome</h4>
										<img class="r-stretch" src="./lib/images/rogdi-mutations.png" alt="rogdi-mutations">
										<br><br>
										<div class='foot-right'>
												Initial KTS genetics: <a href="https://bit.ly/3hMJNMt" target="_blank">Schossig <em>et al.</em>, 2012</a>
								</section>

								<!-- predicting protein subcellular localization -->
								<section data-background-image="lib/images/sv.png" data-background-position="top right" data-background-size="3em">
										<h4>A presynaptic vesicle module containing ROGDI, WDR7, and DMXL2</h4>
										<div class="fig-container r-stretch" data-file="lib/graphs/rogdi/index.html"></div>
								</section>


								<!-- Rogdi-Rav2 alignment -->
								<section>
										<h3>Rogdi, Rav2, and the RAVE Complex</h3>
										<div class="container">
												<div class="col">
														<img src="./lib/images/rogdi-rav2.png" alt="rogdi-structure" style="width:80%">
												</div>
												<div class="col">
														<img src="./lib/images/rogdi-prot-module.png" alt="rogdi-module" style="width:100%">
												</div>
										</div>
										<br>
										<div class='foot-right'>
												Rogdi crystal structure: <a href="https://go.nature.com/3d6jpf1" target="_blank">Lee <em>et al.</em>, 2017</a>
										</div>
								</section>

								<!--
								-->

										</div>
						</div>
						<!-- end slides -->

						<!-- reveal.js src -->
						<script src="dist/reveal.js"></script>
						<script src="plugin/zoom/zoom.js"></script>
						<script src="plugin/math/math.js"></script>
						<script src="plugin/zoom/zoom.js"></script>,
						<script src="plugin/search/search.js"></script>
						<script src="plugin/markdown/markdown.js"></script>,
						<script src="plugin/highlight/highlight.js"></script>
						<script src="plugin/notes/notes.js"></script>
						<script src="plugin/pointer/pointer.js"></script>
						<script>
								Reveal.initialize({
										// slide settings:
										hash: true,
										center: true,
										controls: true,
										progress: true,
										slideNumber: true,
										//backgroundTransition: 'none',
										// plugins:
										plugins: [RevealZoom, RevealNotes, RevealSearch, RevealMarkdown, RevealHighlight, RevealMath, RevealPointer],
										// dependencies: (note: this feature is deprecated!)
										dependencies: [ 
												{ src: 'lib/js/reveald3.js' },
										],
										// RevealPointer configuration:
										pointer: {
												    //key: "q", 
												    color: "#800080", 
												    pointerSize: 16, 
												    //alwaysVisible: false, 
												  }
								});
						</script>

		</body>
</html>
